### PR TITLE
Change package name from 'aad.implementation' to 'aad.webapp'.

### PR DESCRIPTION
**Pull request by chenrujun**
Change package name from 'aad.implementation' to 'aad.webapp'.

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @yiliuTo , please help to review this PR. 🙏

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureClientRegistration.java [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-package com.azure.spring.aad.implementation;
+package com.azure.spring.aad.webapp;
 
 import org.springframework.security.oauth2.client.registration.ClientRegistration;
 
 import java.util.Set;
 
+/**
+ * Azure oauth2 client registration.
+ * It have 2 kind of scopes:
```
Comment (yiliuTo) [inline sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureClientRegistration.java:12]: Should be "It has", but it's not something big deal.

**Review by yiliuTo — APPROVED** [review]
LGTM

File: sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureClientRegistration.java (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AzureClientRegistration.java]: Fixed. Thanks for point out.

**Pull request merged**